.env

```diff
@@ -1,5 +1,9 @@
 REGISTRY=
 WINDOWSSERVERCORE_VERSION=ltsc2019
 NANOSERVER_VERSION=1809
-SITECORE_VERSION=9.2.0
-LICENSE_PATH=c:\license
+SITECORE_VERSION=9.3.0
+SITECORE_LICENSE=
+SQL_SA_PASSWORD=8Tombs-Given-Clock#-arming-Alva-debut-Spine-monica-Normal-Ted-About1-chard-Easily-granddad-5Context!
+TELERIK_ENCRYPTION_KEY=qspJhcSmT5VQSfbZadFfzhCK6Ud7uRoS42Qcm8UofvVLiXciUBcUeZELsTo8KD9o6KderQr9Z8uZ9CHisFJNRz46WTZ5qCRufRFt
```

Comment:
Moeten deze 2 waardes hier wel staan? Qua naam lijkt me iig van niet.

Reply:
Is hetzelfde als het community Sitecore Docker setup: https://github.com/Sitecore/docker-images/blob/master/windows/tests/9.3.x/.env